src/React/README.md

```diff
@@ -0,0 +1,59 @@
+# Symfony UX React
+
+Symfony UX React integrates [React](https://reactjs.org/) into Symfony applications. It provides tools 
+to render React components from Twig.
+
+**This repository is a READ-ONLY sub-tree split**. See
+https://github.com/symfony/ux to create issues or submit pull requests.
+
+## Usage
```

Comment:
It looks like you've got the TODO already, but most of this should be moved into the `index.rst` file.